data/import_data.py

```diff
@@ -0,0 +1,131 @@
+import yaml
+import os
+
+from app import db  # noqa: F401
+from models import (
+    Category,
+    Criterion,
+    HonorableMention,
+    InnovativePolicyIdea,
+    Score,
+    State,
+    StateGrade,
+    StateCategoryGrade,
+    ResourceLink
+)
+from tests.test_utils import clear_database
+
+SCRIPT_DIR = os.path.dirname(__file__)
+
+
+def import_data():
+    print('Clearing database')
+    clear_database(db)
+
+    print('Importing categories')
+    import_categories()
+
+    for filename in os.listdir(absolute_file_path('states')):
+        state = filename.split('.yaml')[0]
+        print(f'Importing state: {state}')
+
+        import_state(absolute_file_path(f'states/{filename}'))
```

Comment:
This is the main body of the import script. It does the following:
1. Clears the database so we don't try to import duplicate data.
2. Imports the scorecard information (categories and criteria)
3. For every file in the `data/states` folder (currently just `AK.yml`), it imports the state data.